.kokoro/docker/Dockerfile

```diff
@@ -0,0 +1,160 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:16.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Ensure local Python is preferred over distribution Python.
+ENV PATH /usr/local/bin:$PATH
+
+# http://bugs.python.org/issue19846
+# At the moment, setting "LANG=C" on a Linux system fundamentally breaks
+# Python 3.
+ENV LANG C.UTF-8
+
+# Install dependencies.
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+    apt-transport-https \
+    build-essential \
+    ca-certificates \
+    curl \
+    git \
+    graphviz \
+    libbz2-dev \
+    libdb5.3-dev \
+    libexpat1-dev \
+    libffi-dev \
+    liblzma-dev \
```

Comment:
Can we add `libmagickwand-dev` here?

Now we install it on the fly:
https://github.com/GoogleCloudPlatform/python-docs-samples/blob/13fb63a93bde73f7394ea9b94f154513f38224ff/.kokoro/tests/run_tests.sh#L58

Reply:
done